parmed/formats/sdf.py

```diff
@@ -27,17 +28,11 @@ def id_format(filename):
         is_fmt : bool
             True if it is a sdf file, False otherwise
         """
-        f = genopen(filename, 'r')
+        line = linecache.getline(filename, lineno=4)
         try:
-            for line in f:
-                if not line.strip(): continue
-                if line.startswith('M  END') or line.startswith('$$$$'):
-                    return True
-                else:
-                    continue
+            return line.split()[-1] in ['V2000', 'V3000']
```

Comment:
seem we can not check the number of words. 

For example: n_words = 8 in http://infochim.u-strasbg.fr/recherche/Download/Fragmentor/MDL_SDF.pdf, page 78; n_words = 10 for the `test.sdf` in this PR.